lib/one.js

```diff
@@ -5,7 +5,7 @@ var files = require('./files');
 var options = require('./args');
 
 module.exports = function one(name, cb){
-    request.get('https://registry.npmjs.org/'+name, {json: true}, function(err, res, body){
+    request.get('https://registry.npmjs.org/'+name.replace('/', '%2F'), {json: true}, function(err, res, body){
```

Comment:
Odd... we need to URI-encode the slash, but not the "at" sign? (Said another way, why not just `encodeURIComponent(name)`?)